packages/yew-macro/src/derive_props/mod.rs

```diff
@@ -79,17 +82,27 @@ impl ToTokens for DerivePropsInput {
         let Self {
             generics,
             props_name,
+            prop_fields,
+            preserved_attrs,
             ..
         } = self;
 
+        for field in prop_fields {
+            match field.ty() {
+                Type::Path(TypePath { qself: None, path }) if is_path_a_string(path) => {
+                    emit_warning!(
+                        path.span(),
+                        "storing string values with `String` is deprecated, use `AttrValue` \
```

Comment:
It's not deprecated, just not recommended. We don't intend to take away the ability to do so. What is deprecated is the implicit conversion from `String` to `AttrValue`, but unfortunately we don't have a way to emit a warning for it.

Can you change the message accordingly? A link to documentation where downsides of using String is mentioned would be great to have as well